File: Python/Iniciante/1009 - Salário com Bônus.py
"""
Autor: Pierre Vieira
Data da submissão: 16/01/2019 23:58:26
"""
nome = input()
salario = float(input())
total_vendas = float(input())
comissao = float(total_vendas*0.15)
print("TOTAL = R$ {:.2f}".format(salario+comissao))

File: C#/Iniciante/1009 - Salário com Bônus.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 13:20:10
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            String nome_vendedor;
            double salario, vendas, comissao;
            nome_vendedor = Console.ReadLine();
            salario = double.Parse(Console.ReadLine());
            vendas = double.Parse(Console.ReadLine());
            comissao = 0.15 * vendas;
            Console.WriteLine($"TOTAL = R$ {(comissao + salario).ToString("F2", CultureInfo.InvariantCulture)}");
        }
    }
}
